newsfragments/7844.doc.rst

```diff
@@ -0,0 +1,4 @@
+Update hosting documentation:
+
+Previously know as the section **Administration** it has been renamed to **Hosting**.
+It's subsection **Hosting** as been renamed to **Deployment**.
```

Comment:
I wouldn't add much detail here about renamed or moved sections. 
For me, as a user, the important thing is that the guide is has been updated to be consistent with V3.
```suggestion
Update hosting guide (previously administration guide) for version 3
```